### PR TITLE
Addition of the holders distribution in academy

### PR DESCRIPTION
**Pull request by spiderjako**

### CODE REVIEW

**Review by antisaa — APPROVED** [review]
LGTM

**Review by tspenov — APPROVED** [review]

File: src/docs/metrics/holders-distribution/index.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+title: Holders Distribution
+author: Santiment Team
+date: 2020-08-24
+---
+
+## Definition
+
+There are five separate metrics regarding holders distribution:
+
+- holders_distribution_<interval> - represents the number of addresses holding the amount of tokens that belong to a given interval
```
Comment (IvanIvanoff) [inline src/docs/metrics/holders-distribution/index.md:11]: You can put the metric names in backticks so they look like this - `holders_distribution_<interval>`

File: src/docs/metrics/holders-distribution/index.md [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+- holders_distribution_combined_balance_<interval> - represents the sum of all the tokens of all the addresses which hold the amount of tokens from an interval
+- holders_distribution_over_<threshold> - the number of addresses whose balance exceeds threshold
```
Comment (IvanIvanoff) [inline src/docs/metrics/holders-distribution/index.md:13]: exceeds `threshold` so it's clearer that it is replaced by the actual value in the metric name

File: src/docs/metrics/holders-distribution/index.md (hunk 1 above)
Comment (IvanIvanoff) [reply src/docs/metrics/holders-distribution/index.md:11]: We also might want to have 2 lists of the available threhsolds and intervals, especially for the intervals as they're not very clear.

**Pull request merged**